src/wallet/wallet.h

```diff
@@ -669,6 +668,39 @@ struct CoinEligibilityFilter
     CoinEligibilityFilter(int conf_mine, int conf_theirs, uint64_t max_ancestors, uint64_t max_descendants) : conf_mine(conf_mine), conf_theirs(conf_theirs), max_ancestors(max_ancestors), max_descendants(max_descendants) {}
 };
 
+struct OutputGroup
+{
+    std::vector<COutput> m_outputs;
+    bool m_from_me;
+    CAmount m_value;
+    int m_depth;
+    size_t m_ancestors;
+    size_t m_descendants;
+    CAmount effective_value;
+    CAmount fee;
+    CAmount long_term_fee;
+
+    OutputGroup() : OutputGroup(std::vector<COutput>(), true, 0, 999, 0, 0) {}
+    OutputGroup(const std::vector<COutput>&& outputs, bool from_me, CAmount value, int depth, size_t ancestors, size_t descendants)
+    : m_outputs(outputs)
+    , m_from_me(from_me)
+    , m_value(value)
+    , m_depth(depth)
+    , m_ancestors(ancestors)
+    , m_descendants(descendants)
+    , effective_value(0)
+    , fee(0)
+    , long_term_fee(0)
```

Comment:
nit: same for all members here. Can just inline into `CAmount long_term_fee{0};` above to avoid duplication.